lib/longshoreman/container.rb

```diff
@@ -76,7 +74,7 @@ def rport(exposed_port, protocol="tcp")
   end
 
   def start
-    @raw.start
+    @raw.start({ 'PublishAllPorts' => true, 'CapAdd' => 'NET_ADMIN' })
```

Comment:
Why?  I know it works with PublishAllPorts in main_args.  Does the CapAdd not work there?